tests/log-in.test.js

```diff
@@ -0,0 +1,43 @@
+import LoginPage from '../pageobjects/login.page'
+
+describe('Banking', () => {
+
+    it(' Should access to homepage with correct username and password ', async () => {
+        await LoginPage.openWeb();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+
+        await expect(await browser.getUrl()).to.equal('http://localhost:8080/bank/home');
+    });
+    
+    it(' Should redirect to homepage when accessing login page with a "remember me" option ', async () => {
+        await LoginPage.openWeb();
+
+        await (await $('#remember-me')).click();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
```

Comment:
You are mixing 2 levels of abstraction here (pageObjects methods and raw web elements located through selectors). Ideally, any function should work at the same level of abstraction to make it easier to mantain and understand. Consider implementing this rememberMe button as a pageObject attribute (the same way you did with username input, password and submit button), and interact with it through it (potentially encapsulating that into a higher-level method such as `clickRememberMe()`). That way you encapsulate all the complexity regarding how to interact with page's elements at pageObject level exclusively (making the process transparent for the test)